helm-charts/mock-bot/templates/ingress.yaml

```diff
@@ -7,7 +7,7 @@ metadata:
   annotations:
     meta.helm.sh/release-name: {{ include "mock-bot.fullname" . }}
     nginx.ingress.kubernetes.io/use-regex: "true"
-    nginx.ingress.kubernetes.io/rewrite-target: /$2
+    nginx.ingress.kubernetes.io/rewrite-target: /client-api/$2
```

Comment:
@decodingahmed is there a better way to achieve this?  I'm trying to lock down to a particular API but couldn't find a way to do it neatly with URL rewriting.  It would be a painful change to adapt if another API were to be required with public routing...

Reply:
I guess that's not too bad.